src/features/instrument/Instrument.tsx

```diff
@@ -21,18 +21,38 @@ const polySynthSquare = new PolySynth(Synth, {
 }).chain(new Volume(-25), Destination);
 
 export const Instrument = () => {
-  // @TODO: separate to piaoa nd predefined isntruments
+  return (
```

Comment:
I am happier with such separation, because:
/let's to concise Instrument component a lot
/Keys and PredefinedMusic are pretty independent, do not rely on parents functions/useRef elements
/Helps to maintain focus in 1 component, since almost all required logic is there
Minus: Predefined Music is pretty large

Do you think it would be better to move `Keys` and `PredefinedMusic` to separate files? What about reusing sampler in both? Pass via props? Create shared utility/constant?